config.json

```diff
@@ -6,14 +6,6 @@
   "deprecated": [
 
   ],
-  "ignored": [
-    "_template",
-    "bin",
-    ".gradle",
-    "docs",
-    "img",
-    "build"
```

Comment:
I've noticed a `build` dir popping up in the exercises directory when running the unit tests, but if that remains an issue after merging this we can squash it separately.